src/scripting/daedalus/DaedalusVMForGameWorld.cpp

```diff
@@ -292,6 +292,24 @@ namespace REGoth
       bs::INT32 bodyTexIndex = popIntValue();
       bs::String bodyMesh    = popStringValue();
 
+      // If an armor is set here, we need to replace the body mesh from the input parameters with the
+      // visual settings from inside the armor instance
+      if (armorInstance != -1)
+      {
+        // TODO: Original Gothic adds the Armor straight to the inventory. We just create a
+        // temporary instance to extract the visual information from it
+        ScriptObjectHandle armorSObj = getInstance(armorInstance);
```

Comment:
`armorInstance` is a `bs::UINT32` but `getInstance` expects a `bs::String`, so it doesn't compile for me due to this line

Reply:
Weird, did I forget to commit something?

Reply:
There seems to be more missing code, actually. Now, I wonder where that went...

Reply:
Can't find it in stashes, it's not on other branches, guess I'lll write that again.

Reply:
Actually I found those commits! Seems like I forgot to push them before merging https://github.com/REGoth-project/REGoth-bs/pull/21...